pkgs/development/tcl-modules/by-name/ji/jimlib/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitLab,
+  mkTclDerivation,
+}:
+
+mkTclDerivation rec {
+  pname = "jimlib";
+  version = "0.16.0";
+
+  src = fetchFromGitLab {
+    owner = "dbohdan";
+    repo = "jimlib";
+    rev = "v${version}";
+    sha256 = "sha256-a7Su/JByGLgnYBCRbP9z9CunTIYTwAisWuQafZWYgYo=";
```

Comment:
It's a SRI hash, so

```suggestion
    hash = "sha256-a7Su/JByGLgnYBCRbP9z9CunTIYTwAisWuQafZWYgYo=";
```

Reply:
Updated and rebased with the accurate version name. Thank you for the help and the patience 😄